### PR TITLE
Fix/systemd log cap and log4j configuration

### PR DESCRIPTION
**Pull request by hcoyote**
1. set a cap on systemd journal 
2. add log4j2.yaml 
3. setup ansible to deploy yaml and modify benchmark-worker

### CODE REVIEW

File: driver-redpanda/deploy/deploy.yaml [hunk 1]
```diff
@@ -66,6 +66,24 @@
...
+    - name: Limit size of journal
+      lineinfile:
+        dest: /etc/systemd/journald.conf
+        regexp: ^SystemMaxUse=
+        line: SystemMaxUse=1000M
+    - name: restart journald
+      systemd:
+        name: systemd-journald
+        state: restarted
+
```
Comment (tmgstevens) [inline driver-redpanda/deploy/deploy.yaml:85]:
```
    - ansible.builtin.lineinfile:
        path: /etc/rsyslog.d/50-default.conf
        insertbefore: '.*syslog'
        line: '$outchannel mysyslog,/var/log/syslog,104857600'
    - ansible.builtin.lineinfile:
        path: /etc/rsyslog.d/50-default.conf
        regexp: '.*syslog'
        line: '*.*;auth,authpriv.none          :omfile:$mysyslog'
    - ansible.builtin.service:
        name: rsyslog
        state: restarted
```

**Comment by tmgstevens** [comment]
My only thought is that we should probably just modify bin/benchmark-worker directly, rather than doing it via Ansible, that way Kafka and other drivers also get to pick it up.

File: driver-redpanda/deploy/deploy.yaml (hunk 1 above)
Comment (hcoyote) [reply driver-redpanda/deploy/deploy.yaml:87]: added below.

**Comment by hcoyote** [comment]
> My only thought is that we should probably just modify bin/benchmark-worker directly, rather than doing it via Ansible, that way Kafka and other drivers also get to pick it up.

Ok, moved the inline edits into the scripts directly. the log4j templates will still be in the redpanda driver templates dir; not sure how to easily work around that right now, as putting them into the benchmark tarball means redeploying the tarball every time you want to make a change.

**Review by tmgstevens — APPROVED** [review]

**Comment by tmgstevens** [comment]
Ideally, I'd like us to get this into https://github.com/redpanda-data/openmessaging-benchmark/blob/main/driver-kafka/deploy/ssd-deployment/deploy.yaml#L383 as well so that the log4j settings get picked up when we do Kafka testing as well. But we can do that as part of a separate change.

**Pull request merged**